torch/nn/functional.py

```diff
@@ -795,6 +809,13 @@ def binary_cross_entropy_with_logits(input, target, weight=None, size_average=Tr
                 over observations for each minibatch. However, if the field
                 sizeAverage is set to False, the losses are instead summed
                 for each minibatch. Default: True
+
```

Comment:
Not sure we need the example in both `functional.py` and in `loss.py` as these functions link to the module docs i.e. `See :class:`~torch.nn.BCEWithLogitsLoss` for details`?